azurelinuxagent/common/protocol/restapi.py

```diff
@@ -68,9 +72,21 @@ class VMAgentManifest(object):
     def __init__(self, family, version=None):
         self.family = family
         # This is the Requested version as specified by the Goal State, it defaults to 0.0.0.0 if not specified in GS
```

Comment:
Should this comment be moved up to where `VERSION_0` is defined?

Reply:
`VERSION_0` is just a constant depicting the `0.0.0.0` value. This comment is specific to the logic specified here so I feel its in the apt place.